docs/basic_training/processes.pt.md

```diff
@@ -0,0 +1,950 @@
+---
+description: Material de treinamento básico do Nextflow
+---
+
+# Processos
+
+No Nextflow, um processo (`process`) é a primitiva de computação básica para executar funções estrangeiras (ou seja, scripts personalizados ou ferramentas).
+
+A definição do processo começa com a palavra-chave `process`, seguida pelo nome do processo e, finalmente, o corpo do processo delimitado por chaves.
+
+O nome do process é comumente escrito em letras maiúsculas por convenção.
```

Comment:
Process inside backticks: O nome do `process` ...

Or

O nome do processo (`process`)

Reply:
Thanks for spotting this one! 😄